src/components/navigation/navigation.tsx

```diff
@@ -227,15 +229,23 @@ export class Navigation extends React.Component<
                       </ChangePlanSelect>
 
                       {this.props.changePlan ? (
-                        <AnchorButton
-                          className="nav-bar-button"
-                          minimal
+                        <Tooltip
+                          content={"Exit Change Planner"}
```

Comment:
To be more consistent... can we say "Exit Change Plan"

Reply:
ur wish is my command